### PR TITLE
switched to default non-masquerade destination ranges

### PR DESCRIPTION
**Pull request by yliaog**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
/kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
switched to default non-masquerade destination ranges

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by lzang** [comment]
lgtm

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 1]
```diff
@@ -1001,12 +1001,8 @@ function Configure_Dockerd_CniNetworking {
   $cidr_range_start = Get_PodIP_Range_Start(${env:POD_CIDR})
 
   # Explanation of the CNI config values:
-  #   CLUSTER_CIDR: the cluster CIDR from which pod CIDRs are allocated.
   #   POD_CIDR: the pod CIDR assigned to this node.
   #   CIDR_RANGE_START: start of the pod CIDR range.
-  #   MGMT_SUBNET: the subnet on which the Windows pods + kubelet will
```
Comment (pjh) [inline cluster/gce/windows/k8s-node-setup.psm1:1007]: Can you clean up the code for `mgmt_subnet` throughout the file? It looks like most/all of it is now unused.

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 2]
```diff
@@ -1069,9 +1073,7 @@ function Configure_Dockerd_CniNetworking {
   replace('DNS_SERVER_IP', ${kube_env}['DNS_SERVER_IP']).`
   replace('DNS_DOMAIN', ${kube_env}['DNS_DOMAIN']).`
   replace('MGMT_IP', ${mgmt_ip}).`
-  replace('CLUSTER_CIDR', ${kube_env}['CLUSTER_IP_RANGE']).`
```
Comment (pjh) [inline cluster/gce/windows/k8s-node-setup.psm1:1072]: Remove CLUSTER_IP_RANGE in the comment at line 987. Same for the containerd function.

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 3]
```diff
@@ -1001,12 +1001,8 @@ function Configure_Dockerd_CniNetworking {
   $cidr_range_start = Get_PodIP_Range_Start(${env:POD_CIDR})
 
   # Explanation of the CNI config values:
```
Comment (pjh) [inline cluster/gce/windows/k8s-node-setup.psm1:1003]: Can you add a comment here explaining that the OutBoundNAT exceptionlist is the CIDRs for which we do not want to perform SNAT, and explain that we exclude all of the non-masquerade destination ranges listed at https://cloud.google.com/kubernetes-engine/docs/how-to/ip-masquerade-agent#default-non-masq-dests ?

Please replicate the comment for the containerd function too.

**Review by pjh — COMMENTED** [review]
Please bring up a cluster manually or make sure our Windows presubmit passes before submitting.

If you bring up a cluster you can also try running the smoke-test.sh script; I think some of the test cases are broken right now due to container image issues, but it might provide a little value...

File: cluster/gce/windows/k8s-node-setup.psm1 (hunk 1 above)
Comment (yliaog) [reply cluster/gce/windows/k8s-node-setup.psm1:1007]: removed.

File: cluster/gce/windows/k8s-node-setup.psm1 (hunk 3 above)
Comment (yliaog) [reply cluster/gce/windows/k8s-node-setup.psm1:1003]: done

File: cluster/gce/windows/k8s-node-setup.psm1 (hunk 1 above)
Comment (yliaog) [reply cluster/gce/windows/k8s-node-setup.psm1:1007]: cleaned up

File: cluster/gce/windows/k8s-node-setup.psm1 (hunk 2 above)
Comment (yliaog) [reply cluster/gce/windows/k8s-node-setup.psm1:1072]: removed

**Pull request merged**